web/src/main/resources/spring/security-ctx.xml

```diff
@@ -0,0 +1,36 @@
+<beans:beans xmlns="http://www.springframework.org/schema/security"
+             xmlns:beans="http://www.springframework.org/schema/beans"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+		                         http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security-4.1.xsd">
+
+    <http create-session="stateless" entry-point-ref="entryPointUnauthorizedHandler">
+        <csrf disabled="true"/>
+        <custom-filter before="FORM_LOGIN_FILTER" ref="authTokenFilter"/>
+
+        <!-- Put the most specific matches at the top of less specific ones -->
```

Comment:
All requests are actually protected (if you don't specify the opposite explicitly):
`<!--<intercept-url pattern="/**" access="isAuthenticated()"/>-->`
But I've commented that line to keep other guys' functionality working. After they implement auth token handling client-side that line will be uncommented.